modified_slot_machine.R

```diff
@@ -0,0 +1,71 @@
+#g et the symbols
+get_symbols <- function(n_wheels = 3) {
```

Comment:
What does (n_wheels = 3) imply here? What changes does this apply to the get_symbols function?

Reply:
It just sets a default condition. Originally I was thinking in the way of "how do you make this scalable"? Your default condition is 3 different slots, but what if you wanted to see how things changed with 4 slots? By supplying it as an argument (the = 3 simply means that "if there is nothing supplied, use this value"), we set the default to use 3 wheels, but keep the option to use more.